version_history.md

```diff
@@ -19,7 +21,51 @@ This file contains archived version history information for the [ROCm project](h
 - [New Features as of ROCm 1.7](#new-features-as-of-rocm-17)
 - [New Features as of ROCm 1.5](#new-features-as-of-rocm-15)
 
+### New features and enhancements in ROCm 2.7.2
 
+This release is a hotfix for ROCm release 2.7.
+
+#### Defect fixed in ROCm 2.7.2
+
+##### A defect in upgrades from older ROCm releases has been fixed.
+
+### New features and enhancements in ROCm 2.7.1
```

Comment:
I would drop this entire section section titled New features and enhancements in ROCm 2.7.1

Reply:
I believe most of the comments in 2.7.1 still apply for 2.7.2

Reply:
Combined